unit_test/fuzzing/test_responder/test_spdm_responder_event_types/event_types.c

```diff
@@ -0,0 +1,104 @@
+/**
+ *  Copyright Notice:
+ *  Copyright 2021-2024 DMTF. All rights reserved.
+ *  License: BSD 3-Clause License. For full text see link: https://github.com/DMTF/libspdm/blob/main/LICENSE.md
+ **/
+
+#include "spdm_unit_fuzzing.h"
+#include "toolchain_harness.h"
+#include "internal/libspdm_responder_lib.h"
+
+#if LIBSPDM_ENABLE_CAPABILITY_EVENT_CAP
+
+size_t libspdm_get_max_buffer_size(void)
+{
+    return LIBSPDM_MAX_SPDM_MSG_SIZE;
+}
+
+void libspdm_test_responder_event_types_case1(void **State)
+{
+    libspdm_test_context_t *spdm_test_context;
+    libspdm_context_t *spdm_context;
+    libspdm_session_info_t *session_info;
+    uint32_t session_id;
+    size_t response_size;
+    uint8_t response[LIBSPDM_MAX_SPDM_MSG_SIZE];
+
+    spdm_test_context = *State;
+    spdm_context = spdm_test_context->spdm_context;
+
+    spdm_context->connection_info.version = SPDM_MESSAGE_VERSION_13 <<
+                                            SPDM_VERSION_NUMBER_SHIFT_BIT;
+    spdm_context->connection_info.connection_state = LIBSPDM_CONNECTION_STATE_NEGOTIATED;
+    spdm_context->response_state = LIBSPDM_RESPONSE_STATE_NORMAL;
+
+    spdm_context->connection_info.capability.flags =
+        SPDM_GET_CAPABILITIES_RESPONSE_FLAGS_EVENT_CAP |
```

Comment:
`SPDM_GET_CAPABILITIES_RESPONSE_FLAGS_EVENT_CAP` is not necessary for requester? right?